app/controllers/lists_controller.rb

```diff
@@ -5,34 +5,34 @@ def index
 
   def new 
     @recipe = Recipe.find(params[:recipe_id])
-    @recipe_list = List.where(recipe_id: @recipe.id).sort_by {|list| list.ingredient_id}
     @list = List.new
   end
 
   def create
     @recipe = Recipe.find(params[:recipe_id])
+    @recipe_list = List.where(recipe_id: @recipe.id).sort_by {|list| list.ingredient_id}
```

Comment:
Line is too long. [88/80]<br>Space between { and | missing.<br>Space missing inside }.